gtsfm/multi_view_optimizer.py

```diff
@@ -24,11 +25,14 @@ def __init__(
         trans_avg_module: TranslationAveragingBase,
         data_association_module: DataAssociation,
         bundle_adjustment_module: BundleAdjustmentOptimizer,
+        use_pose_slam_intitialization: bool = False,
     ) -> None:
+        self.pose_slam_module = PoseSlam()
```

Comment:
nit: It should go into hydra config later?

Reply:
There is no alternative. When we have two versions, yes